active/website.md

```diff
@@ -0,0 +1,61 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
```

Comment:
```suggestion
```

Based on group discussions today, would recommend taking this out so it’s clear the group is meant to focus on features and code, and so we don’t have to figure out how to delegate access to the Django admin.

Reply:
I agree